### PR TITLE
Add option to start an interactive shell

### PR DESCRIPTION
**Pull request by vanadinit**

### CODE REVIEW

File: automatix/command.py [hunk 1]
```diff
@@ -105,9 +113,9 @@ def _execute(self, interactive: bool = False, force: bool = False):
         if self.get_type() == 'manual' or interactive:
             self.env.LOG.debug('Ask for user interaction.')
 
-            answer = self._ask_user(question='[MS] Proceed?', allowed_options=['p', 's', 'R', 'a'])
+            answer = self._ask_user(question='[MS] Proceed?', allowed_options=['p', 'T', 's', 'R', 'a'])
             # answers 'a', 'c' and 'R' are handled by _ask_user, 'p' means just pass
-            if answer == 's':
+            if answer == 's' or self.get_type() == 'manual':
```
Comment (vanadinit) [inline automatix/command.py:118]: Hmm github does not to update Pullrequests, if the main branch changes? This should be already on `main`.

Comment (vanadinit) [reply automatix/command.py:118]: OK, found it. Had to update my branches via "sync fork".

**Comment by MaZderMind** [comment]
LGTM, however I'm unsure what the benefit of opening a local shell from within automatix is compared to just opening a new shell in my terminal emulator of choice. I'm doing that all the time and from my point of view it gives more flexibility and less potential to accidentally kill the automatix process.

**Comment by Ga570n** [comment]
The feature in this pull-request is part of a fork. We will not merge the PR at the moment. So if you are interested in this feature have a look here: https://github.com/vanadinit/automatix_cmd

**Pull request closed without merging**